Formula/anycable-go.rb

```diff
@@ -32,8 +32,7 @@ def install
       "-X github.com/anycable/anycable-go/utils.version=#{version}"
     end
 
-    system "go", "build", "-mod=vendor", *std_go_args(ldflags: ldflags),
-                          "-v", "github.com/anycable/anycable-go/cmd/anycable-go"
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/anycable-go"
```

Comment:
vendor folder got dropped per [this commit](https://github.com/anycable/anycable-go/commit/ba25df95d996a6b2c861268aa4504b018a8a0764)